### ISSUE TITLE
Downloading of files created by assistant.

### ISSUE THREAD

**Issue by gabrielchua** [Chainlit/openai-assistant#3]
Thanks for this template! it's really helpful.

What's the best way to download the files from the assistant, and render it as a download link?

**Comment by dorianturner** [comment]
I do not think that you are able to download files that have been uploaded to the assistant as a safety feature (https://community.openai.com/t/reaccessing-files-uploaded-when-creating-an-assistant/845536)

You may be able to do a rough workaround for downloading files the assistant generates (https://community.openai.com/t/how-do-download-files-generated-in-ai-assistants/493516/3) though I am not sure how this would apply through the chainlit interface
